### DISCUSSION TITLE
property有抛出校验错误，但是formItem没有展示错误信息

### DISCUSSION THREAD

**Discussion by ChimingPhang** [alibaba/formily#3468] [Q&A 1.x]
<img width="557" alt="image" src="https://user-images.githubusercontent.com/3283625/196323773-336f7033-7de6-4c60-a936-f96ea2620d1f.png">
<img width="751" alt="image" src="https://user-images.githubusercontent.com/3283625/196323790-61de018c-546c-4113-8235-8047c829d16d.png">
<img width="732" alt="image" src="https://user-images.githubusercontent.com/3283625/196327686-2ba2c716-f706-41bc-88bf-003616e71fb9.png">

**Comment by frehaiku** [comment]
需传入 FormItem 或其他组件消费 field 模型中 errors 报错
